README.md

```diff
@@ -38,6 +38,19 @@ Add to your `.zshrc`
 export PROMPT="$PROMPT$(git-radar --zsh --fetch) "
 ```
 
+**fish**
+
+Add to your `config.fish`
+```bash
+function fish_prompt
+    set_color $fish_color_cwd
+    echo -n (prompt_pwd)
+    git-radar --bash -fetch
```

Comment:
I know the `--bash` option will work for fish shellers but should we provide a `--fish`, initially as an alias of `--bash`. That way we can provide a native fish prompt in the future without changing the interface? 

All it needs is a change in [`git-radar`](https://github.com/michaeldfallen/git-radar/blob/master/git-radar#L85):
```bash
if [[ "$command" == "--bash" || "$command" == "--fish" ]]; then
  $dot/prompt.bash
fi
```